node/mock/indexerMock.go

```diff
@@ -15,7 +15,7 @@ type IndexerMock struct {
 }
 
 // SaveBlock -
-func (im *IndexerMock) SaveBlock(_ data.BodyHandler, _ data.HeaderHandler, _ map[string]data.TransactionHandler, _ []uint64, _ []string, _ []byte) {
+func (im *IndexerMock) SaveBlock(_ *indexer.ArgsSaveBlockData) {
 	panic("implement me")
```

Comment:
maybe rename this struct to a stub and add those ...Called handlers?

Reply:
done.